docs/services/hpc-console.md

```diff
@@ -0,0 +1,53 @@
+[](){#hpc-console}
+# HPC Console
+
+HCP Console is a web application designed to provide essential functionalities for interacting with [FirecREST API v2 backend][ref-firecrest].
+It provides users with a modern web-based environment to interact with HPC resources such as clusters, job schedulers, and filesystems.
+
+## Features
+
+- A modern web interface built on FirecREST API functionalities.
+- A dashboard offering an overview of configured clusters and their statuses.
+- A dedicated view for creating and monitoring jobs.
+- A file navigator with basic file management functionalities, including upload and download features.
+
+A description of the views and functionalities can be found [here](https://eth-cscs.github.io/firecrest-ui/documentation/).
+
+## Views overview
+
+- Dashboard – shows cluster availability and basic system status.
+- Jobs – allows creating, submitting, and monitoring jobs. Users can provide job name, script, and resource requirements.
+- File Navigator – supports browsing directories, uploading, downloading, and deleting files. Editing files inline is not supported.
```

Comment:
This again kind of repeats what was above, adding a few new details. Do you think it makes sense to consolidate the text a bit?